DESCRIPTION

```diff
@@ -46,6 +46,8 @@ Config/Needs/verdepcheck: insightsengineering/formatters,
     tidyverse/tibble, mllg/checkmate, tidyverse/dplyr, yihui/knitr,
     r-lib/lifecycle, rstudio/rmarkdown, gagolews/stringi, r-lib/testthat,
     r-lib/withr
+Remotes:
+    insightsengineering/formatters@336_nearest_value_round
```

Comment:
once formatter is merged to main, this will change to @main

Reply:
Thanks for flagging this!